corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditor.java

```diff
@@ -198,7 +198,7 @@ public static void printPayload(Map.Entry<CorfuDynamicKey, CorfuDynamicRecord> e
                     .append(JsonFormat.printer().print(entry.getValue().getPayload()));
             System.out.println(builder.toString());
         } catch (Exception e) {
-            //log.error("invalid payload: ", e);
+            //log.info("invalid payload: ", e);
```

Comment:
Not sure why it was commented. Can we log this exception (uncomment)?

Reply:
fixed. thanks